src/mixed_arena.h

```diff
@@ -19,6 +19,7 @@
 
 #include <atomic>
 #include <cassert>
+#include <cstdlib>
```

Comment:
It's needed for `abort`, this just wasn't noticed before because other locations happened to have the include anyhow, I think.